### PR TITLE
Update am-installing.adoc

### PR DESCRIPTION
**Pull request by juanpablotimpanaro**

### CODE REVIEW

File: modules/ROOT/pages/am-installing.adoc [hunk 1]
```diff
@@ -136,7 +136,7 @@ Enabling Monitoring can impact CPU utilization and memory utilization, which var
 
 == What to do Next
 
-- If you find connection failure or retry entries, ensure that ports 443 and 5044 in your outbound firewall are open.
+- If you find connection failure or retry entries, ensure that port 443 in your outbound firewall are open.
```
Comment (sarathecoubian) [inline modules/ROOT/pages/am-installing.adoc:139]:
```suggestion
- If you find connection failure or retry entries, ensure that port 443 in your outbound firewall is open.
```

**Comment by juanpablotimpanaro** [comment]
@michael-lee-mulesoft Old runtimes (RTF, maybe) might still use that old 5044 port. I think it makes sense to add that in the troubleshooting section. Thoughts?

File: modules/ROOT/pages/am-installing.adoc [hunk 2]
```diff
@@ -136,7 +136,7 @@ Enabling Monitoring can impact CPU utilization and memory utilization, which var
 
 == What to do Next
 
-- If you find connection failure or retry entries, ensure that ports 443 and 5044 in your outbound firewall are open.
+- If you find connection failure or retry entries, ensure that port 443 in your outbound firewall is open. Older runtimes might still use port 5044 and that needs to be open as well, if that's the case.
```
Comment (sarathecoubian) [inline modules/ROOT/pages/am-installing.adoc:139]:
```suggestion
- If you find connection failure or retry entries, ensure that port 443 in your outbound firewall is open. If an older runtime uses port 5044, the port must remain open.
```

Comment (sarathecoubian) [reply modules/ROOT/pages/am-installing.adoc:139]: Are connection failure/retry entries log entries? Should specify throughout the topic.

Comment (juanpablotimpanaro) [reply modules/ROOT/pages/am-installing.adoc:139]: Not really. If the runtime is using the old port for some reason, we should mention checking both ports (443/5044) to be open

Comment (juanpablotimpanaro) [reply modules/ROOT/pages/am-installing.adoc:139]: @sarathecoubian Any updates here?

Comment (michael-lee-mulesoft) [reply modules/ROOT/pages/am-installing.adoc:139]: @juanpablotimpanaro Click "commit suggestion".

Comment (sarathecoubian) [reply modules/ROOT/pages/am-installing.adoc]: If that change is okay with you, @juanpablotimpanaro , please commit it. This change has been requested by some users over Slack, so we should make sure it gets merged soon.

**Pull request merged**